lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -145,6 +148,9 @@ export default function ContentSelector({
           listFilesApi={blackboardListFilesApi}
           onCancel={cancelDialog}
           onSelectFile={selectBlackboardFile}
+          // An alias we maintain that provides multiple external documentation links for
+          // different versions of Blackboard (classic vs. ultra)
```

Comment:
```suggestion
          // different versions of Blackboard (Classic vs. Ultra)
```

This is to make it clear that these terms are Blackboard's official names for these versions rather than just descriptions that we use.